charts/service/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: service
 description: A generic k8s service chart
 type: application
-version: 1.7.5
+version: 1.7.7
```

Comment:
Why the jump to 7 and skipping 6?

Reply:
This earlier [PR](https://github.com/codecademy-engineering/helm-charts/pull/96) has 1.7.6.

Reply:
this is a big change, so maybe bump to `1.8.0`

Reply:
Done